components/Tabs/Tabs.js

```diff
@@ -2,61 +2,67 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
     
+    this.element = element;
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data = document.querySelector(`.tabs-link[data-tab='${element.dataset.tab}']`);
```

Comment:
be careful here, you don't want to grab each `.tab-link` element that have the same data-tab value as the current element being passed in, you want to store just the data itself. `this.data` should store `this.element.dataset.tab`